src/common/WebLink.js

```diff
@@ -23,16 +22,10 @@ const componentStyles = createStyleSheet({
 /**
  * A button styled like a web link.
  *
- * Accepts `ZulipText`s, `ZulipTextIntl`s, and strings as children.
+ * Accepts strings as children or a `text` prop, just like ZulipText.
  *
- * Note: This sort of messes in those non-string children's business by
- *   unsetting some of their default style attributes. It does this so that
- *   its own special formatting can be passed down through the limited style
- *   inheritance that RN supports:
- *     https://reactnative.dev/docs/text#limited-style-inheritance
- *   See `aggressiveDefaultStyle` in ZulipText.
- *
- * TODO: Possibly there's a better way handle this.
+ * Also accepts `ZulipText`s and `ZulipTextIntl`s as children. To keep the
+ * special formatting, you have to pass `inheritColor` to those.
```

Comment:
nit: there's also `inheritFontSize`

Reply:
But WebLink doesn't use a `fontSize` attribute as part of its special style, right, which is why I didn't mention it here.

Reply:
Mmmm right. Yeah, agreed.